### PR TITLE
Add nightly builds for current ARM and IBM platforms

### PR DESCRIPTION
**Pull request by kcgen**
This PR adds builds for the ARMv6, ARMv7, AArch64 (64-bit ARM), IBM LinuxONE, and IBM POWER8 platforms using GitHub's [run-on-arch](https://github.com/uraimo/run-on-arch) action.

The action launches a Docker container inside of which the target distribution and platform is run within a QEMU virtual machine.  Although the goal was to have these run along-side our on-push Linux builds, unfortunately the realistic QEMU VM accuracy comes at a steeper time cost versus simply cross-compiling, making these builds more suitable as an out-of-band nightly cron job. 

A preview of what we can expect: https://github.com/dosbox-staging/dosbox-staging/runs/620376059?check_suite_focus=true

Capture
Capture-aarch64
Capture-armv6
Capture-armv7
Capture-linuxone
Capture-power8

### CODE REVIEW

File: scripts/automator/packages/manager-apt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Package repo: https://packages.ubuntu.com/
-packages+=(ccache xvfb libtool build-essential libsdl2-dev libsdl2-net-dev libncurses-dev libopusfile-dev)
+packages+=(ccache libtool build-essential autotools-dev automake libsdl2-dev libsdl2-net-dev libncurses-dev libopusfile-dev)
```
Comment (kcgen) [inline scripts/automator/packages/manager-apt:2]: Annoyingly, `apt` on (at least these tested ARM platforms) does not provide a complete auto-stack with `build-essentials`, so this covers these cases.

**Comment by kcgen** [comment]
Although I didn't include it, we could also count warnings and lift out binaries as assets.
I think the value right now is simply flagging failed builds (catching issues like the recent dr_mp3 typo in ARM-typedef'd code).

File: .github/workflows/platforms.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+name: Platform builds
+
+on:
+  schedule: [cron: '0 14 * * *']
+
+jobs:
+  build_linux_platforms:
+    runs-on: ubuntu-latest
+    name: ${{ matrix.conf.name }}
+    if: github.event_name != 'pull_request' || contains('dreamer,kcgen,ant-222', github.actor) == false
```
Comment (dreamer) [inline .github/workflows/platforms.yml:10]: (just a comment)

In here event will always be `true`, as this is on schedule job, but perhaps this is preparation for running these jobs on push/PR?

**Comment by dreamer** [comment]
> Although I didn't include it, we could also count warnings and lift out binaries as assets.
> I think the value right now is simply flagging failed builds (catching issues like the recent dr_mp3 typo in ARM-typedef'd code).

We can do that in the future PR. This is great base towards closing #2 and also publishing tarballs for RPi builds from our `release/*` branches.

**Comment by dreamer** [comment]
Thanks!

File: .github/workflows/platforms.yml (hunk 2 above)
Comment (kcgen) [reply .github/workflows/platforms.yml:10]: Oh right! That was a copy-and-paste error from when I previously had it in our linux workflow. Feel free to knock that out :-)

**Pull request merged**